test.sh

```diff
@@ -262,7 +262,8 @@ STAGE="gomod-vendor"
 if [[ "${RUN[@]}" =~ "$STAGE" ]] ; then
   print_heading "Running Go Mod Vendor"
   go mod vendor
-  git diff --exit-code
+  run_and_expect_silence git config --global safe.directory "$(pwd)"
```

Comment:
I think setting this git config should be part of build.sh, right?

Reply:
_That_ is a very intriguing idea that didn't occur to me! Giving it a shot now...

Reply:
Yep, that was a great idea, I like this a lot better. Thanks!